web/src/components/logstream/schema.vue

```diff
@@ -868,16 +885,22 @@ export default defineComponent({
         selectedIndices += schema.index_type[i];
       }
 
+      if(selectedIndices.includes('prefixPartition') && option.value.includes('keyPartition')){
+        return true;
+      }
+      if(selectedIndices.includes('keyPartition') && option.value.includes('prefixPartition')){
+        return true;
+      }
       if (
         selectedIndices.includes("hashPartition") &&
         selectedHashPartition !== option.value &&
         (option.value.includes("hashPartition") ||
-          option.value.includes("keyPartition"))
+          option.value.includes("keyPartition") || option.value.includes("prefixPartition"))
+
```

Comment:
as we are doing prefix search this change is not needed because for example if the selectedIndices is hashpartition_8 then it will disable all the options that contains hashPartition in it so we are not implementing this